content/md/products-super-power/enrich-products-variants.md

```diff
@@ -218,6 +218,11 @@ You can add a variant to a product model from the product form via the navigatio
 
 1.  In the navigation component `EU Shoes Size`, click on the `Add new` button, a pop-in is displayed
   ![Add new variant](../img/Products_ProductModel_AddProduct_1.png)
+  
+  ::: ee
```

Comment:
Sorry, could you put it at the end, just before "View their history". Else it creates a break in the actions 1, 2, 3...